app/views/govuk_component/feedback.raw.html.erb

```diff
@@ -0,0 +1,110 @@
+
+        <h2 class="pub-c-feedback__form-heading">Help us improve GOV.UK</h2>
+        <p id="feedback_explanation" class="pub-c-feedback__form-paragraph">Don't include personal or financial information like your National Insurance number or credit card details.</p>
+
+        <div class="pub-c-feedback__form-group js-form-group">
+          <label class="pub-c-feedback__form-label" for="what_doing">
+            What were you doing on this page?
+          </label>
+          <textarea id="what_doing" class="pub-c-feedback__form-field" name="what_doing" rows="4" aria-required="true" aria-describedby="feedback_explanation"></textarea>
+        </div>
+
+        <div class="pub-c-feedback__form-group js-form-group">
+          <label class="pub-c-feedback__form-label" for="what_wrong">
+            What is wrong with this page?
+          </label>
+          <textarea id="what_wrong" class="pub-c-feedback__form-field" name="what_wrong" rows="4" aria-required="true"></textarea>
+        </div>
+
+        <input class="pub-c-feedback__submit" type="submit" value="Submit">
+      </div>
+    </div>
+  </form>
+
+  <form action="/contact/govuk/email-survey-signup" id="page-is-not-useful" class="pub-c-feedback__form pub-c-feedback__form--email pub-c-feedback__js-show js-feedback-form js-hidden" data-track-category="user_satisfaction_survey" data-track-action="banner_taken">
+    <a href="#" class="pub-c-feedback__close js-close-form" aria-controls="page-is-not-useful" aria-expanded="true" role="button">Close</a>
+
+    <div class="pub-c-feedback__grid-row">
+      <div class="pub-c-feedback__column-two-thirds">
+        <div class="pub-c-feedback__error-summary js-hidden js-errors" tabindex="-1"></div>
+
+        <input type="hidden" name="url" value="<%= request.original_url -%>">
+        <input type="hidden" name="user_agent" value="<%= request.user_agent -%>">
+
+        <input name="email_survey_signup[survey_id]" type="hidden" value="user_satisfaction_survey">
+        <input name="email_survey_signup[survey_source]" type="hidden" value="<%= request.original_url -%>">
+        <input name="email_survey_signup[ga_client_id]" type="hidden" value="1627485790.1515403243">
+
+        <h2 class="pub-c-feedback__form-heading">Help us improve GOV.UK</h2>
+        <p id="survey_explanation" class="pub-c-feedback__form-paragraph">To help us improve GOV.UK, we'd like to know more about your visit today. We'll send you a link to a feedback form. It will take only 2 minutes to fill in. Don't worry we won't send you spam or share your email address with anyone.</p>
+
+        <div class="pub-c-feedback__form-group js-form-group">
+          <label class="pub-c-feedback__form-label" for="email-field">Email address</label>
+          <input id="email-field" class="pub-c-feedback__form-field" type="text" name="email_survey_signup[email_address]" aria-describedby="survey_explanation">
```

Comment:
Can this field use https://github.com/alphagov/govuk_publishing_components/pull/152?

Reply:
It can! I didn't know about that. Although it seems like the styling currently isn't working... it looks different in government-frontend's comp guide to static's. Also I think it's supposed to have a border, will investigate.

Reply:
Now added in new PR in govuk_publishing_components.